src/data/init-moves.ts

```diff
@@ -3092,39 +2977,32 @@ export function initMoves() {
     new AttackMove(MoveId.PHOTON_GEYSER, ElementalType.PSYCHIC, MoveCategory.SPECIAL, 100, 100, 5, -1, 0, 7)
       .attr(UseHigherAttackingStatAttr)
       .ignoresAbilities(),
-    /* Unused */
+    // #region USUM Z-Moves (unused)
     new AttackMove(MoveId.LIGHT_THAT_BURNS_THE_SKY, ElementalType.PSYCHIC, MoveCategory.SPECIAL, 200, -1, 1, -1, 0, 7)
       .attr(UseHigherAttackingStatAttr)
```

Comment:
was the unimplemented removed on purpose?

```suggestion
      .unimplemented()
      .attr(UseHigherAttackingStatAttr)
```

Reply:
No, the Z-Moves should still be marked unimplemented. Will fix.